src/Files.App/Helpers/UIFilesystemHelpers.cs

```diff
@@ -144,16 +131,19 @@ public static async Task CutItem(IShellPage associatedInstance)
 
 		public static async Task CopyItem(IShellPage associatedInstance)
 		{
-			DataPackage dataPackage = new DataPackage()
+			var dataPackage = new DataPackage()
 			{
 				RequestedOperation = DataPackageOperation.Copy
 			};
-			ConcurrentBag<IStorageItem> items = new ConcurrentBag<IStorageItem>();
+			ConcurrentBag<IStorageItem> items = new();
 
-			if (associatedInstance.SlimContentPage.IsItemSelected)
+			if (associatedInstance.SlimContentPage.IsItemSelected &&
+				associatedInstance.SlimContentPage.SelectedItems is not null)
```

Comment:
Why did you add this condition? I think it's redundant.
If this is correct, then `CopyItem()` should also be modified.

Reply:
To resolve the null warnings. I can use ! if you think so

Reply:
OK, I understand. Then we should do the same refactoring for `CopyItem()`.

Reply:
Oh, I made a mistake. For `CutItem()`.